### DISCUSSION TITLE
Headings for groups of ingredients in the recipe?

### DISCUSSION THREAD

**Discussion by grilparto** [mealie-recipes/mealie#1870] [Q&A] [answered]
Tried searching a few different ways but didn't find anything.  I have a few recipes that segregate out the ingredients needed for different parts of the recipe.  For example, it might have the ingredients listed for a marinade, then the ingredients listed out for cooking the dish, then ingredients listed for the sauce that goes on it.  It might look something like:

**Marinade Ingredients:**
1 cup this
1/2 cup that
1/4 cup the other

**The Breading Ingredients:**
1 cup this
1/2 cup that
1/4 cup the other

**Sauce Ingredients:**
1 cup this
1/2 cup that
1/4 cup the other

Currently, I've only found that I can create the recipe without any delineation between where the ingredients will be used in the recipe (might end up with duplicate ingredients, creates a little confusion in the recipe) or just add in the headings as ingredients themselves (not always obvious because the textual style doesn't change).

Am I missing the feature if it exists?  Or would this make sense as a feature request (unless one already exists and I missed it)?

**Comment by michael-genson** [comment, marked as answer]
Which version are you on? This is a feature in the 1.0.0 beta:
![image](https://user-images.githubusercontent.com/71845777/204949789-bec8eeba-3f0b-44a5-96d6-95001785080e.png)

**Reply by grilparto** [reply]
Wow that was too obvious.  Yeah, I'm on 1.0.0-beta5.  Thanks for the help.  I also noticed that markdown (at least creating bold text) seems to work on the ingredients as well.
